Clicker/ViewControllers/GroupControlsViewController.swift

```diff
@@ -184,16 +146,46 @@ class GroupControlsViewController: UIViewController {
 extension GroupControlsViewController: ListAdapterDataSource {
 
     func objects(for listAdapter: ListAdapter) -> [ListDiffable] {
+        let numPolls = pollsDateAttendanceArray.reduce(0) { (result, pollsDateAttendanceModel) -> Int in
+            return result + pollsDateAttendanceModel.model.polls.count
+        }
+        
+        infoModel = GroupControlsInfoModel(numMembers: numMembers, numPolls: numPolls, code: session.code)
```

Comment:
do we want to be instantiating these each time the adapter is refreshed? @kevinchan159 @matthewcoufal from what i understand this is called each time `adapter.performUpdates` is called

Reply:
@mathewscullin @mindylou the `SpaceModel` objects should be initialized in `viewDidLoad`, but the other models should stay here in case new members join or the admin toggles the settings for this group

Reply:
if the `infoModel` changes everytime we call `adapter.performUpdates` its fine to leave it here. if not, we should only instantiating it before we call`adapter.performUpdated`